pinot-core/src/main/java/org/apache/pinot/core/transport/QueryServer.java

```diff
@@ -54,6 +60,7 @@
   private EventLoopGroup _bossGroup;
   private EventLoopGroup _workerGroup;
   private Channel _channel;
+  private Class _channelClass;
```

Comment:
It’s better not to use raw types even if you make it a wildcard because it causes IDE warnings.

Reply:
resolved, will push later with other fixes